microsoft-edge/webview2/release-notes/index.md

```diff
@@ -120,6 +123,315 @@ description
 <!-- end of template -->
 
 
+<!-- ====================================================================== -->
+## 1.0.2849.39
+
+Release Date: October 21, 2024
+
+[NuGet package for WebView2 SDK 1.0.2849.39](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.2849.39)
+
+For full API compatibility, this Release version of the WebView2 SDK requires WebView2 Runtime version 130.0.2849.39 or higher.
+
+
+<!-- ------------------------------ -->
+#### Promotions
+
+The following APIs have been promoted to Stable and are now included in this Release SDK.
+
+<!-- ---------- -->
+###### Configure the security warning when saving a file (`SaveFileSecurityCheckStarting` event)
+
+<!--
+promoted to Stable in Oct Release SDK
+promoted from Experimental to Stable in Oct Prerelease SDK
+-->
+
+Added a new `SaveFileSecurityCheckStarting` event.  Your app can register a handler on this event to get the file path, filename extension, and document origin URI information.  You can then apply your own rules to do actions such as the following:
+   * Allow saving the file without presenting a default security-warning UI about the file-type policy.
+   * Cancel the saving.
+   * Create your own UI to manage runtime file-type policies.
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+<!-- todo: equivalent?
+.NET: CoreWebView2SaveFileSecurityCheckStartingEventArgs.GetDeferral Method
+WinRT: CoreWebView2.GetDeferral Property
+Win32: ICoreWebView2SaveFileSecurityCheckStartingEventArgs::GetDeferral
+-->
```

Comment:
Is it correct that the two Oct sections have this inconsistency?

.NET tab: CoreWebView2SaveFileSecurityCheckStartingEventArgs.GetDeferral Method
WinRT tab: CoreWebView2.GetDeferral Property
Win32 tab: ICoreWebView2SaveFileSecurityCheckStartingEventArgs::GetDeferral

@peiche-jessica @victorhuangwq

Reply:
i think Oct sections need to be like orig Experimental section
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/release-notes/?branch=pr-en-us-3279&tabs=winrtcsharp#experimental-apis-4
ie change WinRT tab item from
CoreWebView2.GetDeferral Property
to:
CoreWebView2SaveFileSecurityCheckStartingEventArgs.GetDeferral Method

@peiche-jessica @victorhuangwq - that's my plan

Reply:
done; matched orig Experimental section
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/release-notes/?branch=pr-en-us-3279&tabs=winrtcsharp#experimental-apis-4